website/versioned_docs/version-23.x/SnapshotTesting.md

```diff
@@ -6,7 +6,7 @@ original_id: snapshot-testing
 
```

Comment:
this guide could be unversioned, right? or at least moved entirely to 22

Reply:
inline snapshots came in 23, so shouldn't be part of 22 docs